docs/cpp/configure-intellisense.md

```diff
@@ -0,0 +1,121 @@
+---
+Order: 13
+Area: cpp
+TOCTitle: Configure IntelliSense
+ContentId: 381b7ce1-5766-49b0-ad26-f9eedae70e63
+PageTitle: Configure IntelliSense
```

Comment:
You have a little more room for the PageTitle so perhaps "Configure C/C++ IntelliSense"